### PR TITLE
Transparenz als Standard

### PR DESCRIPTION
**Pull request by koffeinfrei**
Ready for review.

@huerlisi last chance :smile:

### CODE REVIEW

File: source/2015-11-10-transparenz-als-standard.html.markdown.erb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+Vorbild von [Buffer](https://buffer.com/transparency) als expliziten Wert der
+Firma erarbeitet.
+
+Das Ziel dabei war es, zu erörtern, ob eine öffentliche Transparenz für Panter
+als Unternehmen überhaupt Sinn machen würde und wie wir uns diese zu Nutzen
+machen könnten.
+
+<%= image_tag('kalamata-transparenz.jpg') %>
+
+Nach dem Brainstorming an der griechischen Sonne und den unzähligen
```
Comment (exul) [inline source/2015-11-10-transparenz-als-standard.html.markdown.erb]: Sonne und ~~den~~ unzähligen

**Comment by exul** [comment]
:+1: 

Tweet + Link zum Blog und Dashcoard auf panter.ch :smile:

**Comment by toupeira** [comment]
:+1: great article! Maybe add a link to this PR as well? ;-)

**Comment by koffeinfrei** [comment]
> Maybe add a link to this PR as well? ;-)

What do you mean exactly?

File: source/2015-11-10-transparenz-als-standard.html.markdown.erb [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Ein Teil der Daten wird nun schon mittels unserem
+[Dashboard](https://dashboard.panter.ch) in Echtzeit publiziert.
+
+<%= image_tag('dashboard.jpg') %>
+
+Wir sind momentan damit beschäftigt, alle relevanten Daten aufzubereiten und
+diese nach und nach zu veröffentlichen. Das Dashboard wird demzufolge laufend
+erweitert und die Statistiken in zukünftigen Blog-Posts genauer behandelt
+werden.
```
Comment (toupeira) [inline source/2015-11-10-transparenz-als-standard.html.markdown.erb]: @koffeinfrei I meant adding a link in the article, maybe something like this here:

```
Der Code hinter diesem Blog sowie Artikel im Entwurfsstadium sind ebenfalls
[öffentlich auf GitHub zugänglich](https://github.com/panter/blog.panter.ch).
```

Comment (koffeinfrei) [inline source/2015-11-10-transparenz-als-standard.html.markdown.erb]: Ah I see, good idea.

Comment (koffeinfrei) [inline source/2015-11-10-transparenz-als-standard.html.markdown.erb]: Added.

**Pull request merged**